src/components/Layout.tsx

```diff
@@ -1,26 +1,19 @@
-import dynamic from "next/dynamic"
 import Image from "next/image"
 import Link from "next/link"
 import { FC, PropsWithChildren } from "react"
 
+import ConnectWalletButton from "@/components/ConnectWallet/ConnectWalletButton"
 import ConnectWalletModal, {
   DisconnectWalletModal,
 } from "@/components/ConnectWallet/ConnectWalletModal"
 import ExternalLinks from "@/components/ExternalLinks"
+import NetworkSelector from "@/components/NetworkSelector"
 
 import { useConnectWallet } from "@/store/connectWallet"
 
 import FortressBackground from "~/images/fortress-background.gif"
 import FortressLogo from "~/svg/fortress-logo.svg"
 
-const ConnectWalletButton = dynamic(
-  () => import("@/components/ConnectWallet/ConnectWalletButton"),
-  { ssr: false }
-)
-const NetworkSelector = dynamic(() => import("@/components/NetworkSelector"), {
```

Comment:
Hydration errors can occur with components that rely on client state (i.e. wagmi), but these don't need it anymore. Removing dynamic imports allows the server to render the elements on the page, thus reducing layout shift when the client hydrates.